test/nodetool/test_nodetool.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright 2023-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import pytest
+import rest_api_mock
+import json
+import subprocess
+
+
+def test_jmx_compatibility_args(server_context, nodetool):
+    rest_api_mock.set_expected_requests(server_context, [
+        rest_api_mock.request("POST", "/storage_service/keyspace_compaction/system_schema"),
+        rest_api_mock.request("POST", "/storage_service/keyspace_compaction/system_schema"),
+        rest_api_mock.request("POST", "/storage_service/keyspace_compaction/system_schema"),
+        rest_api_mock.request("POST", "/storage_service/keyspace_compaction/system_schema"),
+        rest_api_mock.request("POST", "/storage_service/keyspace_compaction/system_schema"),
+        rest_api_mock.request("POST", "/storage_service/keyspace_compaction/system_schema")])
```

Comment:
Why do these "expected requests" only have the URL but not any of the other parameters which are also important?